setup.py

```diff
@@ -103,8 +121,9 @@
         "mu.debugger",
         "mu.interface",
         "mu.modes.api",
+        "mu.wheels",
     ],
-    python_requires=">=3.5,<3.8",
+    python_requires=">=3.5,<3.9",
```

Comment:
If there are issues installing in 3.8 we should probably revert this before merging to master.

Reply:
This one is tricky. My platform has no issues, and reducing this means I can't build against 3.8 even though it's perfectly possible. FWIW the issues should only be short-term: basically until pgzero produces a public build which doesn't pin pygame<2.0

Reply:
Right, but if it's not going to work in all platform with 3.8, then master would be broken for some people. Can we build in 3.7 until the dependencies are updated for 3.8?